app/hooks/useLocalStorage.ts

```diff
@@ -0,0 +1,20 @@
+import { useState, useEffect } from 'react';
```

Comment:
We already had useLocalStorage. Let's remove this and move the one named useStoredState here.
I have fixed one of the issue with useLocaStorage.